comms/src/peer_manager/mod.rs

```diff
@@ -40,6 +40,10 @@
 //! # use std::sync::Arc;
 //! # use tari_storage::LMDBWrapper;
 //!
+//! # // Remove database files in case they aren't compatible
```

Comment:
Probably just `norun` this test.. I suspect writing to the FS in doc tests will be frowned upon in e.g. docs.rs and other places

Reply:
Agreed